src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -544,10 +522,10 @@ def components(self):
 class IssueTracker(issue_tracker.IssueTracker):
   """Google issue tracker implementation."""
 
-  def __init__(self, project, http_client, config):
+  def __init__(self, project, http_client, component_id):
     self._project = project
     self._client = http_client
-    self._default_component_id = config['default_component_id']
+    self._component_id = component_id
```

Comment:
Can we perhaps keep the `_default` bit here? it makes it a bit clearer what it's meant for: as a fallback for when there's no more testcase-specific component available.

Reply:
Done (I did not know it could be overridden by testcases but that makes sense)